tests/test_traffic_rules.py

```diff
@@ -0,0 +1,67 @@
+"""Test traffic rules API for disabling and enabling traffic rules.
+
+pytest --cov-report term-missing --cov=aiounifi.traffic_rule tests/test_traffic_rules.py
+"""
+
+import pytest
+
+from aiounifi.models.traffic_rule import TrafficRuleEnableRequest
+
+from .fixtures import TRAFFIC_RULES, WIRELESS_CLIENT
+
+
+@pytest.mark.parametrize("is_unifi_os", [True])
+@pytest.mark.parametrize("enable", [True, False])
+async def test_traffic_rule_enable(
+    mock_aioresponse, unifi_controller, unifi_called_with, enable
+):
+    """Test that traffic rule can be enabled and disabled."""
+    mock_aioresponse.put(
+        "https://host:8443/proxy/network/v2/api/site/default/trafficrules/6452cd9b859d5b11aa002ea1",
+        payload={},
+    )
+
+    traffic_rule = TRAFFIC_RULES[0]
+
+    await unifi_controller.request(
+        TrafficRuleEnableRequest.create(traffic_rule, enable)
+    )
```

Comment:
Change this to use the three control methods of Traffic and you're home :)

Reply:
I left this in, but I added additional tests to use the three control methods.

Reply:
Perfect! Will review tomorrow